### PR TITLE
Add new organisation team details

### PR DESCRIPTION
**Pull request by h-lame**
Retire volunteer team, and replace with new organisation team.  We also add the
new group emails for getting in touch about things.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -52,18 +53,25 @@ If you’re still in need of clarification, there are [a number](https://us.pyco
...
 
-If you'd like to join the volunteer team please contact
-[Murray Steele](http://h-lame.com/)
-([@hlame](http://twitter.com/hlame),
-[murray.steele@lrug.org](mailto:murray.steele@lrug.org)) or one of the
-other [volunteers](#volunteer-team).
+ * [James Adam](http://lazyatom.com) ([@lazyatom](https://twitter.com/lazyatom), [james.adam@lrug.org](mailto:james.adam@lrug.org))
+ * [Chris Lowis](http://blog.chrislowis.co.uk/) ([@chrislowis](https://twitter.com/chrislowis), [chris.lowis@lrug.org](mailto:chris.lowis@lrug.org))
+ * [Melinda Seckington](http://missgeeky.com/) ([@mseckington](https://twitter.com/mseckington), [melinda.seckington@lrug.org](mailto:melinda.seckington@lrug.org))
+ * [Tatiana Stantonian](https://github.com/binaryberry) ([@tomstuart](https://twitter.com/binaryberry), [tatiana.stantonian@lrug.org](mailto:tatiana.stantonian@lrug.org))
```
Comment (tomstuart) [inline index.md]: Should be `[@binaryberry](https://twitter.com/binaryberry)` obvs.

File: index.md [hunk 2]
```diff
@@ -52,18 +53,25 @@ If you’re still in need of clarification, there are [a number](https://us.pyco
...
+ * [Murray Steele](http://h-lame.com/) ([@tomstuart](https://twitter.com/hlame), [murray.steele@lrug.org](mailto:murray.steele@lrug.org))
```
Comment (tomstuart) [inline index.md]: Likewise: `[@hlame](https://twitter.com/hlame)`.

**Comment by tomstuart** [comment]
Notwithstanding the above corrections, this change looks good!

File: index.md (hunk 1 above)
Comment (h-lame) [inline index.md]: One of these days I won't need the incredible @tomstuart typo-detection bot to fix things for me...  fixed and rebased, thanks!

File: index.md (hunk 2 above)
Comment (h-lame) [inline index.md]: Likewise...

**Comment by h-lame** [comment]
Why didn't I merge this last week! Oops!

**Pull request merged**